clientjs/rxhtml.js

```diff
@@ -3064,6 +3064,41 @@ var RxHTML = (function () {
     }
   };
 
+  transforms['dateFmt'] = function(format, d) {
+    if (typeof (d) == "string") {
+      // only transform strings
+      if (d == "1-01-01") return ""
+
+      // if datetime - strip out everything after the T
+      var s = d;
+      var k = s.indexOf('T');
+      if (k >= 0) {
+        s = s.substring(0, k);
+      }
+
```

Comment:
It would be better to assume either date or datetime, and I'd do a regex to test for date and if it is then use the regex to break it down into components when I would then turn into a Date object manually. if the regex fails, then I'd parse with the internal Date object (after stripping off the timezone name).

After this step, regardless of the input, you should have a Date or return an "" as it can't be parsed.